packages/common/src/SVGZoomWidget.ts

```diff
@@ -179,9 +179,17 @@ export class SVGZoomWidget extends SVGWidget {
         }
     }
 
-    zoomToFit(transitionDuration?) {
+    zoomToFit(transitionDuration?, bboxScale?: number) {
         if (this._renderElement) {
             const bbox = this._renderElement.node().getBBox();
+            if (bboxScale) {
+                const widthDiff = (bbox.width * bboxScale) - bbox.width;
+                const heightDiff = (bbox.height * bboxScale) - bbox.height;
```

Comment:
bboxScale could be undefined

Reply:
If it's undefined then it will skip that 185 to 192 conditional.

Reply:
Indeed!

Reply:
This just means that zoomToFit would act normally if bboxScale is undefined.